components/atoms/Select/single-select.tsx

```diff
@@ -45,6 +45,20 @@ const SingleSelect = ({
       });
     }
   }, [isOpen, isSearchable]);
+  const truncateLabel = (label: string, wordLimit: number, maxChars: number) => {
+    const words = label.split(" ");
+    let truncated = label;
+    if (words.length > wordLimit) {
+      truncated = words.slice(0, wordLimit).join(" ") + "...";
+    }
+    if (truncated.length > maxChars) {
+      truncated = truncated.slice(0, maxChars - 3) + "...";
+    }
+    return truncated;
+  };
```

Comment:
Do we really need this??.

Same behaviour can be achieved with tailwinds truncate

Reply:
yes i have replaced the function with tailwind truncate.